docs/source/api/apollo-server.mdx

```diff
@@ -330,6 +330,8 @@ An object containing configuration options for connecting Apollo Server to [Apol
 
 Controls whether to allow [Batching Queries](../workflow/requests/#batching) in a single HTTP Request. Defaults to `false`. If a request comes in formatted as an array rather than as a single request object, an error will be thrown ( i.e., `Operation batching disabled`) _unless_ batching is enabled.
 
+> Note: In the case of duplicate response headers from separate requests, the final response's headers will take precedence.
```

Comment:
This is true but only on a header-by-header basis.

Reply:
Ah, good clarification